packages/cli/src/api/fixtures/collect-typescript-jsx/macro.tsx

```diff
@@ -0,0 +1,25 @@
+import { t, defineMessage, plural, Trans } from "@lingui/macro"
+
+// JS Macro usages
+const msg = t`Message`
+
+const withDescription = defineMessage({
+  message: 'Description',
+  comment: "description"
+})
+
+const withTId = t({
+    id: "ID Some",
+    message: "Message with id some"
+  })
+
+// JSX Macro usages
+;<Trans>Hi, my name is {name}</Trans>
+;<Trans context="Context1">Some message</Trans>
+;<span title={t`Title`} />
+;<span
+  title={plural(count, {
+    one: "# book",
+    other: "# books"
+  })}
+/>
```

Comment:
There was no test with macro in CLI package at all, I added one. Here we just generally check that macro is applied. in extractor More macro extraction cases in babel-extractor-plugin